src/client/views/components/todoItem.jsx

```diff
@@ -19,6 +19,14 @@ export default React.createClass({
         if (this.state.inEdit) {
             this.focusOnHeading();
         }
+
+        let el = this.getDOMNode();
+        let isVisible = (el.getBoundingClientRect().top >= 0) && (el.getBoundingClientRect().bottom <= window.innerHeight);
+        if (!isVisible) {
+            setTimeout(() => {
+                window.scrollTo(0,document.body.scrollHeight);
```

Comment:
There's also a `scrollBy` which might work better here (less magic) in that you could work out how much you needed to scroll down by (how much isn't visible) and then scroll!